docs/docs/cmd/entra/m365group/m365group-user-list.mdx

```diff
@@ -16,10 +16,10 @@ m365 entra m365group user list [options]
 
 ```md definition-list
 `-i, --groupId [groupId]`
-: The ID of the Microsoft 365 group. Specify `groupId` or `groupDisplayName` but not both.
+: The ID of the Microsoft 365 group. Specify `groupId` or `groupName` but not both.
 
-`-n, --groupDisplayName [groupDisplayName]`
-: The display name of the Microsoft 365 group. Specify `groupId` or `groupDisplayName` but not both.
+`-n, --groupName [groupName]`
```

Comment:
I would not change this. First of all the previous option naming more aligns to `displayName` we use in commands like `m365group get/remove...`. Second of all it is an unnecessary breaking change which we already have a lot and I would  avoid additional not needed once.

Reply:
Fixed